### ISSUE TITLE
Reproduce BERT on QQP

### ISSUE THREAD

**Issue by eric-haibin-lin** [dmlc/gluon-nlp#514]

**Comment by TaoLv** [comment]
Is this still open?

**Comment by eric-haibin-lin** [comment]
I think the script is fine. We used the script to finetune our BERT base model trained on open web text and got 91.26 test accuracy.

**Comment by TaoLv** [comment]
Still need more code to pre-process the dataset?

**Comment by eric-haibin-lin** [comment]
i don't think so. the current code should work fine. did you hit any problem/error?
